BE/src/interfaces/achievement.ts

```diff
@@ -1,8 +1,10 @@
+import { IStatus } from './status';
+
 export interface IAchievement {
   id: number;
   description: string;
   icon: string;
-  checkComplete: void;
+  checkComplete(s: IStatus): IStatus;
 }
 
 export type getTaskArchiveType = {
```

Comment:
Should have a method type, not an object one, same for other methods from home-work.